code/src/Components/TaskForm.js

```diff
@@ -0,0 +1,84 @@
+import React, {useState, useEffect} from 'react'
+import ThoughtElement from './ThoughtElements';
+import ThoughtForm from './ThoughtForm';
+
+
+
+
+export const TaskForm = () => {
+  const [thought, setThoughts] = useState([]);
+  const [newMessages, setNewMessages] = useState('');
+
+  const handleOnNewThought = (event) => {
+    setNewMessages(event.target.value)
+  }
+
+  useEffect(() => {
+    getThoughts()
+  },[]);
+
+
+  const getThoughts = () => {
+      // setLoading(true);
+      fetch('https://happy-thoughts-technigo.herokuapp.com/thoughts')
+      .then(res => res.json())
+      .then(data => setThoughts(data))
+      // .finally(() => setLoading(false))
+  
+  }
+
+  const handleOnFormSubmit = (event) => {
+    event.preventDefault();
+
+    const options = {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+            message: newMessages
+        })
+      }
+
+    fetch('https://happy-thoughts-technigo.herokuapp.com/thoughts', options)
+    .then(res => res.json())
+    .then(() => getThoughts())
+    .finally(()=> setNewMessages(''))
```

Comment:
The submitting of new messages works well but maybe consider also setting an interval for fetching new thoughts every 5 or so seconds to retrieve thoughts that others might post.